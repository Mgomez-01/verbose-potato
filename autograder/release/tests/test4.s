    SECTION " code", ROM0
    ld a, $36
    ld [$400], a
    rra
    ld [$400], a
    rra
    ld [$400], a
    rra
    ld [$400], a
    rra
    ld [$400], a
    rra
    ld [$400], a
    rra
    ld [$400], a
    rra
    ld [$400], a
    rra
    ld [$400], a
    rra
    ld [$400], a

    jp $400
    halt
