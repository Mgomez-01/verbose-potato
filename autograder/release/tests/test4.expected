36
9B
4D
A6
D3
69
B4
DA
6D
36
TEST EXITED NORMALLY
